sites/public/src/pages/listings.tsx

```diff
@@ -20,31 +19,6 @@ export interface ListingsProps {
   closedListings: ListingWithSourceMetadata[]
 }
 
-const openListings = (listings) => {
-  return listings?.length > 0 ? (
-    <>{getListings(listings)}</>
-  ) : (
-    <div className="notice-block">
-      <h3 className="m-auto text-gray-800">{t("listings.noOpenListings")}</h3>
-    </div>
-  )
-}
-
-const closedListings = (listings) => {
```

Comment:
Is it intended to get rid of the closed listings section at the bottom?

Reply:
yes! on that note I have also removed the now unneeded GET for the closed listings. did not delete the functions though since I believe there are some discussions to show closed listings (but not P0)

Reply:
Great callout! I'm glad we're leaving in the functions, as they likely will add back in closed listings in the future.